packages/ActionMenu/src/ActionMenu.stories.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+
+import Section from '@components/section';
+import readme from '../README.md';
+
+import AddSolid from '@igloo-ui/icons/dist/AddSolid';
+import Delete from '@igloo-ui/icons/dist/Delete';
+import Copy from '@igloo-ui/icons/dist/Copy';
+
+import ActionMenu, { ActionMenuOption } from './ActionMenu';
+import { OptionType } from '../../List/dist/List';
+
+const actionMenuList: ActionMenuOption[] = [
+  {
+    label: 'Add Item',
+    value: 'add',
+  },
+  {
+    label: 'Delete Item',
+    value: 'delete',
+    disabled: true,
+  },
+  {
+    label: 'Copy Item',
+    value: 'copy',
+  },
+];
+
+const actionMenuList2: ActionMenuOption[] = [
+  {
+    label: 'Add Item',
+    value: 'add',
+    icon: <AddSolid size="medium" />,
+  },
+  {
+    label: 'Delete Item',
+    value: 'delete',
+    icon: <Delete size="medium" />,
+  },
+  {
+    label: 'Copy Item',
+    value: 'copy',
+    icon: <Copy size="medium" />,
+  },
+];
+
+export default {
+  title: 'Components/ActionMenu',
+  component: ActionMenu,
+  parameters: {
+    description: readme,
+  },
+  argTypes: {
+    closeOnSelect: {
+      control: { type: 'boolean' },
+    },
+  },
+} as ComponentMeta<typeof ActionMenu>;
+
+const Template: ComponentStory<typeof ActionMenu> = (args) => (
+  <ActionMenu {...args} style={{ display: 'flex', justifyContent: 'center' }} />
+);
+export const Overview = Template.bind({});
+Overview.args = {
+  options: actionMenuList,
+};
+
+export const Positioning = () => {
```

Comment:
It would be interesting to have them open by default for testing in chromatic and for the user. Also we should display the indication of their position.